app/models/library_entry.rb

```diff
@@ -12,8 +13,10 @@
 #  rating          :decimal(2, 1)
 #  reconsume_count :integer          default(0), not null
 #  reconsuming     :boolean          default(FALSE), not null
+#  started_at      :datetime
 #  status          :integer          not null, indexed => [user_id]
 #  volumes_owned   :integer          default(0), not null
+#  consumed_at      :datetime
```

Comment:
`last_progress_at` or `progress_updated_at` or `progressed_at` might be more clear.